### PR TITLE
update app timeout to 15 min

### PR DESCRIPTION
**Pull request by bbodine1**
## Summary by CodeRabbit

- **New Features**
  - Extended the inactivity timeout period from 10 seconds to 15 minutes to improve user experience during longer periods of inactivity.

- **Documentation**
  - Updated documentation to reflect the new inactivity timeout settings across different environments.

### CODE REVIEW

**Review by kristinmerbach — DISMISSED** [review]

File: apps/provider-gateway/src/environments/environment.uat.ts [hunk 1]
```diff
@@ -4,8 +4,8 @@ const gatewayApiUrl = 'https://api-provider.dbh.dc.gov';
 const portalApiUrl = 'https://api-portal.dbh.dc.gov';
 const authority = `https://${subdomain}.b2clogin.com/${subdomain}.onmicrosoft.com/b2c_1_sign_in_1`;
 const authorityDomain = `${subdomain}.b2clogin.com`;
-// const appInactiveTimeout = 900; // 15 minutes
-const appInactiveTimeout = 10; // 10 seconds
+const appInactiveTimeout = 900; // 15 minutes
+// const appInactiveTimeout = 10; // 10 seconds
```
[the remark this answers is not in the document]
Comment (bbodine1) [reply apps/provider-gateway/src/environments/environment.uat.ts:8]: Keeping this comment for future debugging purposes

File: apps/provider-gateway/src/environments/environment.prod.ts [hunk 2]
```diff
@@ -4,8 +4,8 @@ const gatewayApiUrl = 'https://provider-gateway-api.dbh.dc.gov';
 const portalApiUrl = 'https://ite-api.dbh.dc.gov';
 const authority = `https://${subdomain}.b2clogin.com/${subdomain}.onmicrosoft.com/b2c_1_sign_in_1`;
 const authorityDomain = `${subdomain}.b2clogin.com`;
-// const appInactiveTimeout = 900; // 15 minutes
-const appInactiveTimeout = 10; // 10 seconds
+const appInactiveTimeout = 900; // 15 minutes
+// const appInactiveTimeout = 10; // 10 seconds
```
[the remark this answers is not in the document]
Comment (bbodine1) [reply apps/provider-gateway/src/environments/environment.prod.ts:8]: Keeping this comment for future debugging purposes

File: apps/provider-gateway/src/environments/environment.ts [hunk 3]
```diff
@@ -8,8 +8,8 @@ const gatewayApiUrl = 'https://api.provider.dev.dbhite.com';
 const portalApiUrl = 'https://api.portal.dev.dbhite.com';
 const authority = `https://${subdomain}.b2clogin.com/${subdomain}.onmicrosoft.com/b2c_1_sign_in_1`;
 const authorityDomain = `${subdomain}.b2clogin.com`;
-// const appInactiveTimeout = 900; // 15 minutes
-const appInactiveTimeout = 10; // 10 seconds
+const appInactiveTimeout = 900; // 15 minutes
+// const appInactiveTimeout = 10; // 10 seconds
```
[the remark this answers is not in the document]
Comment (bbodine1) [reply apps/provider-gateway/src/environments/environment.ts:12]: Keeping this comment for future debugging purposes

File: apps/provider-gateway/src/environments/environment.uat.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (bbodine1) [reply apps/provider-gateway/src/environments/environment.uat.ts:8]: Keeping this comment for future debugging purposes

**Review by kristinmerbach — APPROVED** [review]

**Pull request merged**